docs/setting-up-environment.md

```diff
@@ -0,0 +1,85 @@
+# Setting up local run environment
+
+This is useful for development and debugging purposes.
+
+All commands below are provided for IntelliJ IDEA running on Ubuntu. Prior to running them, clone the repository and open in IntelliJ IDEA.
+
+To learn more about some variables referenced here, see [instructions to run the pipeline on the cluster](import/running-the-import.md)
+
+## 1. Install and configure Java 8
+By default, IDEA will use Java 11 on most modern systems. However, this project can only run on Java 8. If you don't have it, install it:
+ ```bash
+ sudo apt install openjdk-8-jdk openjdk-8-jre
+```
+
+In IDEA, open File → Project Structure. Change the “Project SDK” to Java 8.
+
+## 2. Configure Maven
+* Open File → Settings → Build, Execution, Deployment → Build Tools → Maven and change the location of the “User settings file” to `settings.xml` in the root of the repository. To change the setting, you will need to enable the “Override” checkbox on the right.
+* In the `settings.xml` file, fill in `ORACLE_SSO_USERNAME` and `ORACLE_SSO_PASSWORD` (see above).
+* In IDEA, right click on the project → Maven → Reimport. Wait for the process to complete.
+* Right click on the project → Maven → Generate sources and update folders.
+
+## 3. Create and configure local Postgres database
+If necessary, install and enable Postgres installation:
```

Comment:
It would be good to give the option to use docker as well with some command like
```
docker run --name postgres -p 5432:5432 -e POSTGRES_PASSWORD=pass -d postgres
```

Reply:
Thanks, a good point! Addressed in https://github.com/EBIvariation/metadata-ws/pull/86/commits/8b5ca5ac8341bc5c655f664fe2776dfd67037331.